psi4/src/psi4/dlpno/dlpno.h

```diff
@@ -26,21 +26,35 @@
  * @END LICENSE
  */
 
-#ifndef PSI4_SRC_DLPNO_MP2_H_
-#define PSI4_SRC_DLPNO_MP2_H_
+#ifndef PSI4_SRC_DLPNO_H_
+#define PSI4_SRC_DLPNO_H_
 
 #include "sparse.h"
 
 #include "psi4/libmints/wavefunction.h"
+#include "psi4/libmints/matrix.h"
+#include "psi4/libmints/vector.h"
+#include "psi4/libqt/qt.h"
+#include "psi4/libpsio/psio.h"
+#include "psi4/psifiles.h"
+
+#include <map>
+#include <tuple>
+#include <string>
+#include <unordered_map>
 
 namespace psi {
 namespace dlpno {
 
+enum AlgorithmType { MP2, CCSD, CCSD_T };
```

Comment:
Any reason to use an `enum` here instead of an `enum class`?

Reply:
Made change to `enum class`